### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by phoet**
this is the first draft of pulling out the librato metrics from https://github.com/thumbor/thumbor/pull/521

i could not figure out though how i could reference one of the commits from that branch within `setup.py` maybe someone with real python skills can help here :smile_cat: 

/cc @heynemann @masom @seichner

### CODE REVIEW

**Comment by heynemann** [comment]
Looks good! :+1:

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name='tc_librato',
+    version="0.0.1",
+    description='Thumbor Librato extensions',
+    author='Peter Schröder, Sebastian Eichner',
+    author_email='peter.schroeder@jimdo.com, sebastian.eichner@jimdo.com',
+    zip_safe=False,
+    include_package_data=True,
+    packages=find_packages(),
+    install_requires=[
+        'jimdo-thumbor',
```
Comment (phoet) [inline setup.py:15]: @heynemann i was trying to reference 23a400b903da78aa9b65f91a1e6231fb294e9a3d from `Jimdo/thumbor` with something like 

``` python
    dependency_links=[
        "git://github.com/Jimdo/thumbor.git@23a400b903da78aa9b65f91a1e6231fb294e9a3d#egg=thumbor-jimdo"
    ],
```

but i could not get it working. any ideas?

Comment (masom) [inline setup.py:15]: `pip` can install a specific git version in `requirements.txt` but I'm not sure about setup.py

Comment (phoet) [inline setup.py:15]: @masom yeah, i tried that and it works. from what i've read `setup.py` has removed support for this.

**Pull request merged**